### PR TITLE
Update music-api version

### PR DESCRIPTION
**Pull request by little-buddy**
According to this issue

https://github.com/Binaryify/NeteaseCloudMusicApi/issues/1388#issuecomment-962372728

If it is a `flac file`, the download file is in `flac` format.

### CODE REVIEW

**Comment by little-buddy** [comment]
测试歌曲支持 无损音乐


image

请求参数
image


<br/>

|  浏览器获取的    |  登录状态非会员通过postman 获取    |
| ---- | ---- |
|   image   |    image|










结论: 目前 网易云Api 非会员最高能拿到 极高 品质的音乐

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "packages/api"]
+	path = packages/api
+	url = git@github.com:little-buddy/NeteaseCloudMusicApi.git
```
Comment (Linkontoask) [inline .gitmodules]: What is the difference between the warehouse and https://github.com/Binaryify/NeteaseCloudMusicApi

Comment (little-buddy) [reply .gitmodules]: Merging your previously customized code, 
Now you can use this library to upgrade latest without affecting `package.scripts`.

Comment (Linkontoask) [reply .gitmodules]: Can we put the warehouse under this organization

Comment (Linkontoask) [reply .gitmodules]: https://github.com/radishes-music/NeteaseCloudMusicApi

Comment (Linkontoask) [reply .gitmodules]: @little-buddy Permissions and repositories are done, modify the direction of the submodule

**Pull request merged**